tests/docker/util.sh

```
@@ -184,14 +184,15 @@ function prepare_next_gen_columnar_data_dirs() {
 
 function check_env() {
   local cur_dir=$(pwd)
-  local prebuilt_bin_dir=$(realpath "${cur_dir}/../../tests/.build/tiflash")
-  if [[ ! -d ${prebuilt_bin_dir} ]]; then
-    echo "No pre-build tiflash binary directory: ${prebuilt_bin_dir}"
+  local tiflash_bin_dir="${LOCAL_TiFLASH_BIN_DIR:-${cur_dir}/../.build/tiflash}"
+  tiflash_bin_dir="$(realpath "${tiflash_bin_dir}")"
+  if [[ ! -d ${tiflash_bin_dir} ]]; then
+    echo "No TiFlash binary directory: ${tiflash_bin_dir}"
     exit -1
```

Shellcheck (0.11.0)

[error] Can only exit with status 0-255. Other data should be written to stdout/stderr.

(SC2242)